subsys/bluetooth/shell/l2cap.c

```diff
@@ -183,11 +236,25 @@ static int cmd_register(const struct shell *shell, size_t argc, char *argv[])
 		server.sec_level = strtoul(argv[2], NULL, 10);
 	}
 
+	if (argc > 3) {
+		policy = argv[3];
+
+		if (!strcmp(policy, "whitelist")) {
```

Comment:
you can make `whitelist` and `16byte_key` a subcommands. It will simplify `cmd_register` implementation. What is more these subcommands can be autocompleted.

Reply:
I would leave it as it is